README.md

```diff
@@ -194,3 +194,10 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 ```
+
+
+# Command: `populate-workflows.sh`
+
+This repository also includes a shell script, `populate-workflows.sh`, for
+automatically creating or updating the sample workflows shown above in a
+repository.  Run it with `./populate-workflows.sh path/to/local/repo/clone`.
```

Comment:
we might later just remove above snippet whenever templated one populated one by script diverges